finesse/gui/serial_view.py

```diff
@@ -14,7 +16,6 @@ def __init__(
         self,
         devices: Dict[str, Dict[str, str]],
         avail_ports: Sequence[str],
-        avail_baud_rates: Sequence[str],
```

Comment:
I would not remove this, but rather set as default value the imported `BAUDRATES`. That way, the result is more functional, with the functionality depending entirely on the inputs and not on constants defined somewhere else.

Reply:
Good point. Not sure why I didn't do this in the first place.